fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -13,6 +13,8 @@ def self.run(params)
         Spaceship::Tunes.select_team
         UI.message("Login successful")
 
+        platform = "ios"
```

Comment:
Is this safe to hard code this here? Should we provide and option that defaults to 'ios' but could be 'tvOS', 'watchOS', etc?

Reply:
In attempting to provide the option the cascading impact was more broad that I was confident to address. I may try again.

Reply:
Yeah, +1 to @ohayon, adding an option to: `    def self.available_options` with a default of `:ios` would be rad.

Reply:
I've added the platform option to both app_store_build_number and latest_testflight_build_number and tested that the default value is applied for both cases (for iOS)